apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -1852,6 +1852,9 @@
         {% if console.enable_identity_claims is defined %}
         "enableIdentityClaims": {{ console.enable_identity_claims }},
         {% endif %}
+        {% if scim2.custom_user_schema_uri is defined %}
```

Comment:
Why are we adding this to console deployment config even though it's not being used in the console app?

Reply:
Usage: https://github.com/wso2/identity-apps/pull/7213/files#diff-d7687dc53dbafb6c7c2e39bb220798613f42f1e6c7540f52db7d76e63a1f9c30R361